bin/prelude.h

```diff
@@ -24,6 +24,9 @@ predicate u_character(unsigned char *p; unsigned char c);
 predicate integer(int *p; int v);
 predicate u_integer(unsigned int *p; unsigned int v);
 
+predicate long_integer(long long *p; long long l);
+predicate u_long_integer(unsigned long long *p; unsigned long long l);

```

Comment:
Wouldn't it make more sense to call these `llong_integer` and `u_llong_integer`?